tests/ludwig/models/modules/test_encoder.py

```diff
@@ -171,3 +196,108 @@ def test_image_encoders_stacked_2dcnn():
         output_shape=output_shape,
         output_data=None
     )
+
+
+def test_sequence_encoder_embed():
+    num_sentences = 4
+    embedding_size = 5
+    max_len = 6
+
+    # Generate data
+    text, vocab = generate_random_sentences(
+        num_sentences=num_sentences,
+        max_len=max_len,
+    )
+
+    encoder_args = {'embedding_size': embedding_size, 'vocab': vocab}
+
+    # Different values for reduce_output and the corresponding expected size
+    reduce_outputs = ['sum', None, 'concat']
+    output_shapes = [
+        [num_sentences, embedding_size],
+        [num_sentences, max_len, embedding_size],
+        [num_sentences, max_len * embedding_size]
+    ]
+
+    for reduce_output, output_shape in zip(reduce_outputs, output_shapes):
+        for trainable in [True, False]:
+
+            encoder_args['reduce_output'] = reduce_output
+            encoder_args['embeddings_trainable'] = trainable
+            encoder = create_encoder(EmbedEncoder, encoder_args)
+
+            encoder_test(
+                encoder=encoder,
+                input_data=text,
+                regularizer=L1_REGULARIZER,
+                dropout_rate=DROPOUT_RATE,
+                output_dtype=np.float,
+                output_shape=output_shape,
+                output_data=None
+            )
+
+            embed = encoder.embed_sequence.embed
+            assert embed.representation == 'dense'
+            assert embed.embeddings_trainable == trainable
+            assert embed.regularize is True
+            assert embed.dropout is False
+
+
+def test_sequence_encoders():
```

Comment:
Add a test for unhappy paths - bad encoder parameter initialization and make sure they return errors?